### ISSUE TITLE
fission fn logs produces no output

### ISSUE THREAD

**Issue by soamvasani** [fission/fission#1053]
via slack -- `fission fn logs` for a python function showed no logs even though the function is definitely logging.  This shows up in the fluentd pod's logs:

```
2019-01-11 22:13:27 +0000 [warn]: #0 record_reformer: unknown placeholder `${kubernetes_labels_functionUid}` found
```

**Comment by life1347** [comment]
duplicate #625

**Comment by soamvasani** [comment]
Closed by #1086
